buildcontainer/Makefile

```diff
@@ -95,10 +105,33 @@ manifest-push:
 	@echo "pushing manifest $(IMAGE_NAME)"
 	docker manifest push $(IMAGE_NAME)
 
-.PHONY: tags
-tags:
-	@echo $(IMAGE_NAME) $(foreach arch,$(SUBIMAGES), $(IMAGE_NAME)-$(arch))
-
-.PHONY: tag
-tag:
-	@echo $(TAG_VERSION)
+.PHONY: turbo-cross-%
+turbo-cross-%:
+	@echo "building for $(@:turbo-cross-%=%)"
+	docker build \
+		--platform=linux/$(@:turbo-cross-%=%) \
+		--build-arg BASE_IMAGE=$(IMAGE_NAME) \
+		--build-arg GIT_REPO=$(GIT_REPO) \
+		-t $(TURBO_IMAGE_NAME)-$(@:turbo-cross-%=%) \
+		-f Dockerfile.turbo .
+
+.PHONY: turbo-cross
+turbo-cross: $(patsubst %, turbo-cross-%,$(SUBIMAGES))
+
+.PHONY: push-turbo-cross-%
+push-turbo-cross-%: turbo-cross-%
+	docker push $(TURBO_IMAGE_NAME)-$(@:push-turbo-cross-%=%)
+
+.PHONY: push-turbo-cross
+push-turbo-cross: $(patsubst %, push-turbo-cross-%,$(SUBIMAGES))
+
+.PHONY: manifest-create-turbo
+manifest-create-turbo:
+	@echo "creating manifest $(TURBO_IMAGE_NAME)"
+	docker manifest rm $(TURBO_IMAGE_NAME) 2> /dev/null || true
+	docker manifest create $(TURBO_IMAGE_NAME) $(foreach arch,$(SUBIMAGES), --amend $(TURBO_IMAGE_NAME)-$(arch))
+
+.PHONY: manifest-push-turbo
+manifest-push-turbo:
+	@echo "pushing manifest $(TURBO_IMAGE_NAME)"
+	docker manifest push $(TURBO_IMAGE_NAME)
```

Comment:
Some comments above each task would be great for folks who don't know Make that well (i.e. me 😅 )